instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/SemconvStability.java

```diff
@@ -78,4 +78,10 @@ public static String stableDbSystemName(String oldDbSystem) {
   }
 
   private SemconvStability() {}
+
+  public static void setForTesting(
```

Comment:
Imo this is a rather risky approach, as this can cause unexpected results when multiple test classes are run together.

Reply:
I don't like it either - I just didn't find a way to set sys props for a test suite

Reply:
> I don't like it either - I just didn't find a way to set sys props for a test suite

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/1e251627e250af2f5b71fd0b33ff546b3ea5a6b0/instrumentation/aws-sdk/aws-sdk-1.11/javaagent/build.gradle.kts#L111